### PR TITLE
[Validator] Add missing translations for for Thai (th)

### LINKED ISSUE #51958: Missing translations for Thai (th)
Hello,

There are some missing Thai (th) translations and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Thai (th) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.th.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.th.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by parinz1234**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? |no 
| Issues        | Fix #51958 
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.th.xlf [hunk 1]
```diff
@@ -402,6 +402,30 @@
...
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>ค่านี้ประกอบด้วยตัวอักษรที่ไม่รับอนุญาตจากระดับข้อบังคับปัจจุบัน</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>ไม่อนุญาตให้ใช้ตัวอักษรที่มองไม่เห็น</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>ไม่อนุญาตให้ผสมตัวเลขจากสคริปต์ที่แตกต่างกัน</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.th.xlf:423]: GPT suggests `ไม่อนุญาตให้ผสมตัวเลขจากระบบเขียนที่แตกต่างกัน`, would that make sense to you?

Comment (parinz1234) [reply src/Symfony/Component/Validator/Resources/translations/validators.th.xlf:423]: Thanks for fast response, I think GPT try to translates `script` to `ระบบเขียน` for me this word is rarely used. I think it's ok to use **Transliteration** for `script`.

Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.th.xlf:423]: great, thanks for the confirmation

**Comment by nicolas-grekas** [comment]
Thank you @parinz1234.

**Pull request merged**